nautobot_ssot/integrations/aristacv/utils/nautobot.py

```diff
@@ -26,20 +210,20 @@ def verify_site(site_name):
     try:
         site_obj = Location.objects.get(name=site_name, location_type=loc_type)
     except Location.DoesNotExist:
-        site_obj = Location(
+        status, _ = Status.objects.get_or_create(name="Staging")
```

Comment:
We should probably be checking if this was created and adding ContentTypes to the Location object. Although it should exist since it's there by default it's possible a User removes it for some reason and breaks this part.

Reply:
Could you pls. add a suggestion? Adding content type to the location type is on line 209.